gammapy/image/profile.py

```diff
@@ -254,7 +254,7 @@ class ImageProfile(object):
     def __init__(self, table):
         self.table = table
 
-    def smooth(self, kernel="box", radius=0.1 * u.deg, **kwargs):
+    def smooth(self, kernel="box", radius="0.1 deg", **kwargs):
```

Comment:
What happens if somebody tries to pass a quantity for the radius? does it still work as expected?

Reply:
Yes, passing in a `Quantity` still works the same way. The string argument just offers additional convenience to the user. Since Astropy 2.0 it is possible to instantiate quantities with strings,  which I find nicer to use as creating the quantity with `0.1 * u.deg`.

Reply:
👍

Reply:
We should update the dependencies in the setup.py to astropy 2.0 then.

Reply:
Done in this PR see https://github.com/gammapy/gammapy/pull/1898/commits/6813e08a2a5da893eb610cb25e2bae8e48fb43d5.